core/services/ocr2/delegate.go

```diff
@@ -582,6 +583,7 @@ func (d *Delegate) newServicesGenericPlugin(
 		ContractTransmitter:          provider.ContractTransmitter(),
 		ContractConfigTracker:        provider.ContractConfigTracker(),
 		OffchainConfigDigester:       provider.OffchainConfigDigester(),
+		MetricsRegisterer:            prometheus.DefaultRegisterer,
```

Comment:
I wonder whether we should use https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#WrapRegistererWith here and add e.g. the job name as a label to make it easier to query these metrics?
Ditto in lots of other places.

Reply:
Added in [`0a6b80f` (#11954)](https://github.com/smartcontractkit/chainlink/pull/11954/commits/0a6b80f6c2089c147cfb44e31138dc7a46258c07)